### PR TITLE
Add Methodology section of transportation electrification

### PR DESCRIPTION
**Pull request by rouille**
[Pull Request doc](https://breakthrough-energy.github.io/docs/user/git_guide.html#d-pull-request)

### Purpose
Migrate documentation on transportation electrification in the open source docs

### What the code is doing
N/A

### Testing
N/A

### Where to look
Diagrams and tables. Otherwise tplots and text have been copied/pasted. Note that the bibliography is enclosed in a file (refs.bib) in the **docs** repository (within the `ben/transportation`). Before mergin `transportation_electrification` branch into `develop`, the companion branch in **docs** will have to be merged

### Usage Example/Visuals
Below is a PDF of the PreTREISE part. Check out the methodology section and the bibliography at the end of the document
[6. PreREISE — documentation.pdf](https://github.com/Breakthrough-Energy/PreREISE/files/10502533/6.PreREISE.documentation.pdf)

### Time estimate
20min

### CODE REVIEW

File: docs/demand/transportation_electrification/methodology.rst [hunk 1]
```diff
@@ -0,0 +1,761 @@
...
+          B(Input: census population data <br> for urban area and states)
+          C(Input: DoT Transportation and Health Tool's <br> VMT per capita for urban areas and states)
+          B------C
+        end
+        A====input
+        D(Convert state-level VMT per capita into one <br> year's total of urban and rural driving for each state)
+        input==>D
+        subgraph middle[ ]
+          direction LR
+          E(Annual amount of VMT is distributed <br> acroos weeks and months based on <br> driving data from EPA's MOVES)
```
Comment (BainanXia) [inline docs/demand/transportation_electrification/methodology.rst]: `acroos` -> `across`

File: docs/demand/transportation_electrification/methodology.rst [hunk 2]
```diff
@@ -0,0 +1,761 @@
...
+        direction TB
+        subgraph input[ ]
+          direction RL
+          A(Base-year and simulation-year <br> profile of BEV trips)
+          B(Input: NREL's EFS for fuel efficiency <br> projections by vehicle type)
+          B---->A
+         end
+         C(Fuel efficiency projections determine <br> charging needed after each vehicle trip)
+        input==>C
+        D(Chrhing algorithm: immediate and smart)
```
Comment (BainanXia) [inline docs/demand/transportation_electrification/methodology.rst]: `Chrhing` -> `Charging`

File: docs/demand/transportation_electrification/methodology.rst [hunk 3]
```diff
@@ -0,0 +1,761 @@
...
+.. _ldv_smart_charging_output:
+
+.. figure:: demand/transportation_electrification/img/methodology/ldv_smart_charging_output.png
+   :align: center
+
+   Normalized LDV smart charging output for 168 hours (1 week)
+
+.. _ldt_smart_charging_output:
+
+.. figure:: demand/transportation_electrification/img/methodology/ldt_smart_charging_output.png
```
Comment (BainanXia) [inline docs/demand/transportation_electrification/methodology.rst:257]: For some reason, there is a gray border for this figure but not observed in others.

File: docs/demand/transportation_electrification/methodology.rst [hunk 4]
```diff
@@ -0,0 +1,761 @@
...
+resulting charging profile for each day type is replicated across the year, i.e., each
+weekday and each weekend are the same set of trips across the year.
+:numref:`trip_count_table` presents the total number of trips in the trip datasets for
+each vehicle category. The weekday and weekend trips are weighted based on the MOVES
+weight factors. The charging demand is scaled up and down further based on the MOVES
+monthly weight factors depending on the month of the year.
+
+.. _trip_count_table:
+
+.. table:: Trip count by vehicle category, census civision, and day of week
```
Comment (BainanXia) [inline docs/demand/transportation_electrification/methodology.rst]: `civision` -> `division`

File: docs/demand/transportation_electrification/methodology.rst [hunk 5]
```diff
@@ -0,0 +1,761 @@
...
+.. _weekday_vs_weekend_weight_table:
+
+.. table:: Weekday versus weekend weight factors
+
+    +------------------------------+---------+----------+
+    | Day Type                     | Rural   | Urban    |
+    +==============================+=========+==========+
+    | Weekday (divided over 5 days | 0.72118 | 0.762365 |
+    +------------------------------+---------+----------+
+    | Weekend (divided over 2 days | 0.27882 | 0.237635 |
```
Comment (BainanXia) [inline docs/demand/transportation_electrification/methodology.rst]: Right bracket is missing.

File: docs/demand/transportation_electrification/methodology.rst [hunk 6]
```diff
@@ -0,0 +1,761 @@
...
+| :math:`n` is the total number of dwell per trip periods
+
+The total number of dwell segments :math:`seg(i)` will depend on the how long the
+vehicle is parked.
+
+**Equality** constraint:
+
+.. math::
+
+    \sum_{i=1}^n \sum_{j=1}^{seg(i)} x_{ij} + \sum_{i=1}^n y_i
```
Comment (BainanXia) [inline docs/demand/transportation_electrification/methodology.rst]: Equal sign is missing in this equality constraint. Should `+` be `=` instead?

File: docs/demand/transportation_electrification/methodology.rst [hunk 7]
```diff
@@ -0,0 +1,761 @@
...
+| :math:`\Delta t` is the dwell time in [h]
+| :math:`\eta` is the charging efficiency
+
+The optimization is structured to minimize the cost to charge each battery electric
+vehicle within defined battery constraints. It is conducted at hourly timescale,
+meaning that cost and electricity demand are provided in hourly segments. Charging
+efficiency is dependent on the type of electric vehicle supply equipment (EVSE).
+Efficiency tends to increase with higher charging rates.
+
+f a dwell time (:math:`\Delta t`) falls below one hour, the charge (x) available for
```
Comment (BainanXia) [inline docs/demand/transportation_electrification/methodology.rst]: `f` -> `If`

File: docs/demand/transportation_electrification/methodology.rst (hunk 3 above)
Comment (rouille) [reply docs/demand/transportation_electrification/methodology.rst:257]: This is the same figure than in the original documentation. I have the source code so I can try to remove the background

File: docs/demand/transportation_electrification/methodology.rst (hunk 6 above)
Comment (rouille) [reply docs/demand/transportation_electrification/methodology.rst]: Good catch. It should be equal to 0

File: docs/demand/transportation_electrification/methodology.rst (hunk 7 above)
Comment (rouille) [reply docs/demand/transportation_electrification/methodology.rst]: Done

File: docs/demand/transportation_electrification/methodology.rst (hunk 6 above)
Comment (rouille) [reply docs/demand/transportation_electrification/methodology.rst]: Done

File: docs/demand/transportation_electrification/methodology.rst (hunk 5 above)
Comment (rouille) [reply docs/demand/transportation_electrification/methodology.rst]: Done

File: docs/demand/transportation_electrification/methodology.rst (hunk 4 above)
Comment (rouille) [reply docs/demand/transportation_electrification/methodology.rst]: Done

File: docs/demand/transportation_electrification/methodology.rst (hunk 2 above)
Comment (rouille) [reply docs/demand/transportation_electrification/methodology.rst]: Done

File: docs/demand/transportation_electrification/methodology.rst (hunk 1 above)
Comment (rouille) [reply docs/demand/transportation_electrification/methodology.rst]: Done

**Review by BainanXia — APPROVED** [review]
🙏

File: docs/demand/transportation_electrification/methodology.rst (hunk 3 above)
Comment (rouille) [reply docs/demand/transportation_electrification/methodology.rst:257]: Done

**Pull request merged**